hackathon_site/dashboard/frontend/src/components/general/Footer/Footer.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Footer from "./Footer";
+
+describe("Footer", () => {
+    test("renders", () => {
+        const { asFragment } = render(<Footer />);
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    test("date", () => {
+        const fakeDate = new Date("2021-05-17");
+        const fakeYear = "2021";
+
+        const fakeDateClass = class extends Date {
```

Comment:
An easier way to do all of the date mocking is:
```javascript
const fakeYear = 2021;
global.Date.getFullYear = jest.fn(() => fakeYear);
```

With that, you can remove this class and the `global.Date = fakeDateClass`, and the `fakeDate` object above.

Rather than mocking the entire date class, this keeps things simple by mocking just the one function this component needs.